### ISSUE TITLE
build failure under CMake daily build

### ISSUE THREAD

**Issue by meshula** [PixarAnimationStudios/OpenSubdiv#1201]
In order to build with Xcode 12.2, the upcoming CMake 3.19 is required to generate correct Xcode project files.

CMake now treats a condition as an error that was previously not an error. Specifically, when generating frameworks for MacOS, and when a shared library is requested, both frameworks are created to the same path, due to:

```
        set_target_properties(
            osd_static_framework
            PROPERTIES
            FRAMEWORK true
            INSTALL_NAME_DIR "@rpath/OpenSubdiv.framework/OpenSubdiv"
            INSTALL_RPATH "@executable_path/Frameworks;@loader_path/Frameworks"
            OUTPUT_NAME OpenSubdiv
            CLEAN_DIRECT_OUTPUT true
        )

and

            set_target_properties(
                osd_dynamic_framework
                PROPERTIES
                RPATH true
                FRAMEWORK true
                INSTALL_NAME_DIR "@rpath/OpenSubdiv.framework/OpenSubdiv"
                INSTALL_RPATH "@executable_path/Frameworks;@loader_path/Frameworks"
                OUTPUT_NAME OpenSubdiv
                CLEAN_DIRECT_OUTPUT true
            )

```

CMake now treats it as an error that two targets have the same output. An easy fix is to change the OUTPUT_NAME of the shared library to be unique, e.g. OpenSubdiv_shared, or to enforce that either a shared or static library may be built but not both. It would seem least intrusive and bothersome to simply adopt a new name.

**Comment by jilliene** [comment]
Filed as internal issue #OSD-331

**Comment by davidgyu** [comment]
Fixed by #1214
